README.md

```diff
@@ -49,6 +60,12 @@ For more details about the SDK check the links below:
 | low_level_example | <a href="https://github.com/analogdevicesinc/ToF/tree/master/bindings/python/examples/low_level_example"> Python</a> | A simple example of how to get access to the low-level API of the camera. |
 | ROS2 CPP Wrapper | <a href="https://github.com/analogdevicesinc/ToF/tree/master/bindings/ros2/"> ROS2/C++</a> | ROS2 binding that publishes topics |
 
+## Other Examples
+| Example | Language | Description |
+| --------- | ------------- | ----------- |
+| ROS2 Application | <a href="https://github.com/analogdevicesinc/adi_3dtof_adtf31xx"> C++ </a> | A more extensive ROS2 example based on the ADI ToF SDK. |
+| Stitching Algorithm | <a href="https://github.com/analogdevicesinc/adi_3dtof_image_stitching"> C++ </a> | A stiching algorithm using ADI ToF data. |
```

Comment:
Should we mention that this algorithm is also made with ROS2?

Reply:
The stitching algo seems to be ROS Neotic - which seems to be ROS1.

Reply:
ROS2 will come in mid-December or so. They may continue ROS1 support, so I will leave the landing page for the algo to state what is supported.